mne/io/pick.py

```diff
@@ -226,10 +226,10 @@ def pick_types(info, meg=True, eeg=False, stim=False, eog=False, ecg=False,
         If True include ECG channels.
     emg : bool
         If True include EMG channels.
-    ref_meg: bool | str
+    ref_meg : bool | str
         If True include CTF / 4D reference channels. If 'auto', the reference
```

Comment:
`If True, include`...

Reply:
There are a ton of these in this file, so I'm not going to change this one for now as it will make the review process worse. Let's fix it in a follow-up PR.